### PR TITLE
Minor edits to address review comments for the MPI Operator proposal.

### PR DESCRIPTION
**Pull request by rongou**
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/community/140)

### CODE REVIEW

File: proposals/mpi-operator-proposal.md [hunk 1]
```diff
@@ -78,6 +83,15 @@ spec:
       containers:
       - name: tensorflow-benchmarks
         image: rongou/tensorflow_benchmarks:latest
```
Comment (gaocegege) [inline proposals/mpi-operator-proposal.md]: Hi, where could we find the source code of the image?

File: proposals/mpi-operator-proposal.md [hunk 2]
```diff
@@ -12,7 +12,9 @@ multi-node training using TensorFlow; the TensorFlow team is also working on an
 [implementation](https://github.com/tensorflow/tensorflow/tree/master/tensorflow/contrib/distribute).
```
Comment (gaocegege) [inline proposals/mpi-operator-proposal.md]: Could we add a link to http://mpitutorial.com/tutorials/mpi-reduce-and-allreduce/ for the introduction to allreduce here:

```diff
- with each other directly via the MPI `allreduce` primitive
+ with each other directly via the MPI [`allreduce`](http://mpitutorial.com/tutorials/mpi-reduce-and-allreduce/) primitive
```

File: proposals/mpi-operator-proposal.md (hunk 1 above)
Comment (rongou) [reply proposals/mpi-operator-proposal.md]: Source code forthcoming, which would include the Dockerfile to build this image.

Comment (gaocegege) [reply proposals/mpi-operator-proposal.md]: Thanks :clap:

File: proposals/mpi-operator-proposal.md (hunk 2 above)
Comment (rongou) [reply proposals/mpi-operator-proposal.md]: Sure.

**Pull request merged**